bundles/automation/pom.xml

```diff
@@ -20,6 +20,7 @@
     <module>org.eclipse.smarthome.automation.commands</module>
     <module>org.eclipse.smarthome.automation.parser.gson</module>
     <module>org.eclipse.smarthome.automation.providers</module>
+	<module>org.eclipse.smarthome.automation.provider.file</module>
```

Comment:
could you please replace the tab by spaces (or alternatively all spaces by tabs)?